packages/@react-aria/list/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@react-aria/list",
+  "version": "3.0.0-alpha.1",
```

Comment:
Removed the private since it was failing lint (published package uses this). Lemme know if I should bump this version to 3.0.0